lib/bigtable/row.js

```diff
@@ -0,0 +1,745 @@
+ *           ]
+ *         }
+ *       ]
+ *     }
+ *   }, {
+ *     commitRow: true
+ *   }
+ * ];
+ *
+ * Row.formatChunks_(chunks);
+ * // {
+ * //   follows: {
+ * //     gwashington: [
+ * //       {
+ * //         value: 2
+ * //       }
+ * //     ]
+ * //   }
+ * // }
+ */
+Row.formatChunks_ = function(chunks) {
+  var families = [];
+  var chunkList = [];
+
+  chunks.forEach(function(chunk) {
+    if (chunk.resetRow) {
+      chunkList = [];
+    }
+
+    if (chunk.rowContents) {
+      chunkList.push(chunk.rowContents);
+    }
+
+    if (chunk.commitRow) {
+      families = families.concat(chunkList);
+      chunkList = [];
+    }
+  });
+
+  return Row.formatFamilies_(families);
+};
+
+/**
+ * Formats a rowContents object into friendly format.
+ *
+ * @private
+ *
+ * @param {object[]} families - The row families.
+ *
+ * @example
+ * var families = [
+ *   {
+ *     name: 'follows',
+ *     columns: [
+ *       {
+ *         qualifier: 'gwashington',
+ *         cells: [
+ *           {
+ *             value: 2
+ *           }
+ *         ]
+ *       }
+ *     ]
+ *   }
+ * ];
+ *
+ * Row.formatFamilies_(families);
+ * // {
+ * //   follows: {
+ * //     gwashington: [
+ * //       {
+ * //         value: 2
+ * //       }
+ * //     ]
+ * //   }
+ * // }
+ */
+Row.formatFamilies_ = function(families) {
+  var data = {};
+
+  families.forEach(function(family) {
+    var familyData = data[family.name] = {};
+
+    family.columns.forEach(function(column) {
+      var qualifier = Mutation.convertFromBytes(column.qualifier);
+
+      familyData[qualifier] = column.cells.map(function(cell) {
+        return {
+          value: Mutation.convertFromBytes(cell.value),
+          timestamp: cell.timestampMicros,
+          labels: cell.labels
+        };
+      });
+    });
+  });
+
+  return data;
+};
+
+/**
+ * Create a new row in your table.
+ *
+ * @param {object=} entry - An entry. See {module:bigtable/table#insert}.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this
+ *     request.
+ * @param {module:bigtable/row} callback.row - The newly created row object.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * var callback = function(err, apiResponse) {
+ *   if (!err) {
+ *     // Row successfully created
+ *   }
+ * };
+ *
+ * row.create(callback);
+ *
+ * //-
+ * // Optionally, you can supply entry data.
+ * //-
+ * row.create({
+ *   follows: {
+ *      alincoln: 1
+ *   }
+ * }, callback);
+ */
+Row.prototype.create = function(entry, callback) {
+  var self = this;
+
+  if (is.function(entry)) {
+    callback = entry;
+    entry = {};
+  }
+
+  entry = {
+    key: this.id,
+    data: entry,
+    method: Mutation.methods.INSERT
+  };
+
+  this.parent.mutate(entry, function(err, apiResponse) {
+    if (err) {
+      callback(err, null, apiResponse);
+      return;
+    }
+
+    callback(null, self, apiResponse);
```

Comment:
Double checked the proto files, can confirm all it returns is a list of statuses.